### ISSUE TITLE
Add titles to embedded iframes (YouTube)

### ISSUE THREAD

**Issue by atsansone** [flutter/website#9528]
Add `title` attribute to all `iframe` tags throughout the site. These most commonly appear with YouTube video embeds. The iframes without this attribute violates WCAG [H64](https://www.w3.org/TR/WCAG20-TECHS/H64).

**Error:** Iframe element requires a non-empty title attribute that identifies the frame.
[htmlcs: WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1](https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/#sc2_4_1)

**Example:**
```html
<iframe style="max-width: 100%; width: 100%; height: 230px;" src="https://youtube.com/embed/xWV71C2kp38" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```

**Location:**
(selector "html > body > div:nth-child(6) > div > main > div > div:nth-child(13) > div:nth-child(1) > div > iframe")
